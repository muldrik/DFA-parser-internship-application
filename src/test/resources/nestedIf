a = 10
if a > 1 if a > 2 if a > 3 if a > 4 if a > 5 if a > 6
b = a
end end end end end end
a = 20