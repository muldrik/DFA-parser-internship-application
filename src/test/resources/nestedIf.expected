b = a
a = 20
